.github/workflows/release_js_api.yml

```diff
@@ -163,4 +164,4 @@ jobs:
           tag_name: js-api/v${{ needs.build.outputs.version }}
           draft: false
           prerelease: ${{ needs.build.outputs.prerelease == 'true' }}
-          generate_release_notes: true
```

Comment:
I would like to keep this, if possible. These release notes will tag contributors to the release, they will be shown.

It's important to show credits to people

Reply:
`generate_release_notes` generates also the short changelog no?